templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -965,7 +965,33 @@ export default {
         if (!validPreviousAnswers) {
           errMsgs.push("Please select a previous activity to display")
         }
-
+        if (
+          this.node.typeData.activity.questions[0].answerTypes.text.allowMultiple
+        ) {
+          let listQuestion = this.node.typeData.activity.questions[0].answerTypes
+            .text.list
+          let minValue = parseInt(listQuestion.minFields, 10)
+          if (!Number.isInteger(minValue)) {
+            errMsgs.push("Minimum number of fields must be an integer value")
+          }
+          if (minValue < 1) {
+            errMsgs.push("Minimum number of fields for list activity must be >= 1")
+          }
+          if (listQuestion.maxFields.enabled) {
+            let maxValue = parseInt(listQuestion.maxFields.value, 10)
+            if (!Number.isInteger(maxValue)) {
+              errMsgs.push("Maximum number of fields must be an integer value")
+            }
+            if (maxValue < minValue) {
+              errMsgs.push(
+                "Maximum number of fields for list activity must be >= minimum number fields"
+              )
+            }
+            if (maxValue > 100) {
+              errMsgs.push("Maximum number of fields must be <= 100")
```

Comment:
I think we should make these messages more user friendly, without <= signs or the words "fields" and "integer.
"Minimum number of fields must be an integer value" can be "Please enter a number as the minimum number of items"